oak-run/src/main/java/org/apache/jackrabbit/oak/run/MetricsExporterFixtureProvider.java

```diff
@@ -98,7 +102,14 @@ static class ExportMetricsArgs {
             this.pushUri = split.get(1);
 
             if (split.size() > 2) {
-                this.pushMap = Splitter.on(",").omitEmptyStrings().trimResults().withKeyValueSeparator("=").split(split.get(2));
+                this.pushMap = Arrays.stream(split.get(2).split(","))
+                        .map(String::trim)
+                        .filter(s -> !s.isEmpty())
+                        .map(s -> s.split("=", 2))
+                        .filter(arr -> arr.length == 2)
+                        .collect(Collectors.toMap(
+                                arr -> arr[0],
+                                arr -> arr[1]));
```

Comment:
As with MemoryDocumentStore I'm concerned about a possible change in the error handling

Reply:
This one is called from `DataStoreCommand` class and we are catching `Throwable` and logging it.